README.md

```diff
@@ -1223,6 +1223,25 @@ Other Style Guides
     foo.init();
     ```
 
+  <a name="modules--multiline-imports-over-newlines"></a><a name="10.8"></a>
+  - [10.8](#modules--multiline-imports-over-newlines) Multi-line imports should be preferred over single line
```

Comment:
this isn't accurate. It's totally fine to use multiple imports on a single line - the section we're looking for is simply to say that *if* the import is across more than one line, how it should be indented.